docs/sources/send-data/k8s-monitoring-helm/_index.md

```diff
@@ -0,0 +1,304 @@
+* **Storage**:  In this tutorial, Loki will use the default object storage backend provided in the Loki Helm chart; [MinIO](https://min.io/docs/minio/kubernetes/upstream/index.html). You should migrate to a more production-ready storage backend like [S3](https://aws.amazon.com/s3/getting-started/), [GCS](https://cloud.google.com/storage/docs), [Azure Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/) or a MinIO Cluster for production use cases.
+
+<!-- INTERACTIVE ignore START -->
+## Prerequisites
+
+Before you begin, you will need the following:
+
+* A Kubernetes cluster running version `1.23` or later.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your local machine.
+* [Helm](https://helm.sh/docs/intro/install/) installed on your local machine.
+
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Kubernetes Monitoring with Loki](https://killercoda.com/grafana-labs/course/loki/k8s-monitoring-helm).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/loki/loki-ile.svg)
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
+{{< /admonition >}}
+<!-- INTERACTIVE ignore END -->
+
+<!-- INTERACTIVE page intro.md END -->
+
+<!-- INTERACTIVE page step1.md START -->
+
+## Step 1: Create the `meta` and `prod` namespaces
+
+The K8s Monitoring Helm chart will monitor two namespaces: `meta` and `prod`:
+- `meta` namespace: This namespace will be used to deploy Loki, Grafana, and Alloy.
+- `prod` namespace: This namespace will be used to deploy the sample application that will generate logs.
+
+Create the `meta` and `prod` namespaces by running the following command:
+
+```bash
+kubectl create namespace meta && kubectl create namespace prod
+```
+
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Step 2: Add the Grafana Helm repository
+
+All three Helm charts (Loki, Grafana, and the Kubernetes Monitoring Helm) are available in the Grafana Helm repository. Add the Grafana Helm repository by running the following command:
+
+```bash
+helm repo add grafana https://grafana.github.io/helm-charts && helm repo update
+```
+
+It's recommended to also run `helm repo update` to ensure you have the latest version of the charts.
+
+## Step 3: Clone the tutorial repository
+
+Clone the tutorial repository by running the following command:
+
+```bash
+git clone https://github.com/grafana/alloy-scenarios.git && cd alloy-scenarios/k8s-logs
+```
+
+As well as cloning the repository, we have also changed directories to `alloy-scenarios/k8s-logs`. **The rest of this tutorial assumes you are in this directory.**
```

Comment:
At this point in the sandbox, I am NOT in alloy-scenarios/k8s-logs.  I did a little snooping around to find the right directory, but this needs to be fixed.

Reply:
hmm, that is odd the above command should clone the repo and then move your terminal into that directly with the 
`cd alloy-scenarios/k8s-logs`. I shall separate out the commands :)